### PR TITLE
Updated testing to Go 1.11.4 & 1.10.7

### PR DESCRIPTION
**Pull request by MichaelS11**
Added/updated non-English string tests

### CODE REVIEW

**Comment by MichaelS11** [comment]
@mattn Good to go

File: oci8SqlString_test.go [hunk 1]
```diff
@@ -184,16 +184,16 @@ func TestSelectDualString(t *testing.T) {
 			results: [][]interface{}{[]interface{}{strings.Repeat("ab", 1000)}},
 		},
 		testQueryResult{
-			args:    []interface{}{"こんにちは 世界 Καλημέρα κόσμε こんにちは안녕하세요góðan dagGrüßgotthyvää päivääyá'át'ééhΓεια σαςВiтаюგამარჯობაनमस्ते你好здравейсвят"},
-			results: [][]interface{}{[]interface{}{"こんにちは 世界 Καλημέρα κόσμε こんにちは안녕하세요góðan dagGrüßgotthyvää päivääyá'át'ééhΓεια σαςВiтаюგამარჯობაनमस्ते你好здравейсвят"}},
+			args:    []interface{}{"こんにちは 世界 Καλημέρα κόσμε こんにちは안녕하세요góðan dagGrüßgotthyvää päivääyá'át'ééhΓεια σαςВiтаюგამარჯობაनमस्ते你好"},
+			results: [][]interface{}{[]interface{}{"こんにちは 世界 Καλημέρα κόσμε こんにちは안녕하세요góðan dagGrüßgotthyvää päivääyá'át'ééhΓεια σαςВiтаюგამარჯობაनमस्ते你好"}},
```
Comment (mattn) [inline oci8SqlString_test.go]: I can't read this but is this change expected?

Comment (MichaelS11) [reply oci8SqlString_test.go]: Yeah. I can not read it either. Just adding more characters to make sure they work.

Comment (mattn) [reply oci8SqlString_test.go]: :+1:

**Comment by mattn** [comment]
Thanks!

**Pull request merged**